### ISSUE TITLE
Can't use period='*' on CMIP6 if the archive contains a file '*-clim.nc'

### ISSUE THREAD

**Issue by jservonnat** [senesis/climaf#109]
When using period='*', ds() will find all the files with, for instance, _YYYYMM-YYYYMM.nc, but also the files with _YYYYMM-YYYYMM-clim.nc.

**Comment by jservonnat** [comment]
Here is an example:
```
req = ds(project='CMIP6', variable='*p*', experiment='historical', model='*', period='*',
         version='latest', table='*mon')

req.explore('choices')
```
error    : 'Cannot find a period in /bdd/CMIP6/CMIP/IPSL/IPSL-CM6A-LR/historical/r1i1p1f1/Amon/pfull/gr/latest/pfull_Amon_IPSL-CM6A-LR_historical_r1i1p1f1_gr_185001-201412-clim.nc with regexp /bdd/CMIP6/.*/.*/.*/historical/r1i1p1f.*/.*mon/.*p.*/g.*/latest/.*p.*_.*mon_.*_historical_r1i1p1f.*_g.*_(?P<period>(?P<start>[0-9]{4}((01|02|03|04|05|06|07|08|09|10|11|12)(([0-3][0-9])((00|01|02|03|04|05|06|07|08|09|10|11|12|13|14|15|16|17|18|19|20|21|22|23)([0-5][0-9])?)?)?)?)([_-](?P<end>[0-9]{4}((01|02|03|04|05|06|07|08|09|10|11|12)(([0-3][0-9])((00|01|02|03|04|05|06|07|08|09|10|11|12|13|14|15|16|17|18|19|20|21|22|23)([0-5][0-9])?)?)?)?))?).nc'

**Comment by senesis** [comment]
solved with version 1.2.9
